examples/svelte-apollo/src/Articles.svelte

```diff
@@ -0,0 +1,44 @@
+<script context="module">
+  import { gql } from '@apollo/client'
+  import { client } from './apollo'
+
+  export const ARTICLES = gql`
+    {
+      article {
```

Comment:
this query would have to be `articles` if the table name is changed to `articles`.

Reply:
Done